futures-util/src/stream/try_stream/mod.rs

```diff
@@ -76,6 +78,18 @@ mod try_filter;
 #[allow(unreachable_pub)] // https://github.com/rust-lang/rust/issues/57411
 pub use self::try_filter::TryFilter;
 
+mod try_forward;
+
+#[cfg(feature = "sink")]
+delegate_all!(
+    /// Future for the [`try_forward`](super::TryStreamExt::try_forward) method.
+    #[cfg_attr(docsrs, doc(cfg(feature = "sink")))]
+    TryForward<St, Si>(
+        try_forward::TryForward<St, Si, St::Item>
```

Comment:
This is the source of the compile error.
```suggestion
        try_forward::TryForward<St, Si, St::Ok>
```

Reply:
Aha, I left that from the previous changes.